index.html

```diff
@@ -24,8 +24,12 @@
 </script>
 <body>
     <div id="header"></div>
-    <div id="hero"></div>
+    <div id="hero" class="">
+        
+    </div>
```

Comment:
Remove this unused `class` attribute instead of living it empty! And remove this unused space on line 28
```suggestion
    <div id="hero"></div>
```